app/app.go

```diff
@@ -734,14 +734,22 @@ func (app *App) setPostHanders() {
 // governance.
 func (*App) BlockedParams() [][2]string {
 	return [][2]string{
+		// MaxSquareSize
+		{blobmoduletypes.ModuleName, string(blobmoduletypes.KeyGovMaxSquareSize)},
 		// bank.SendEnabled
 		{banktypes.ModuleName, string(banktypes.KeySendEnabled)},
-		// staking.UnbondingTime
-		{stakingtypes.ModuleName, string(stakingtypes.KeyUnbondingTime)},
-		// staking.BondDenom
-		{stakingtypes.ModuleName, string(stakingtypes.KeyBondDenom)},
+		// MaxBlockBytes and consensus.block.TimeIotaMs
+		{baseapp.Paramspace, string(baseapp.ParamStoreKeyBlockParams)},
 		// consensus.validator.PubKeyTypes
 		{baseapp.Paramspace, string(baseapp.ParamStoreKeyValidatorParams)},
+		// consensus.Version.AppVersion
+		{baseapp.Paramspace, string(baseapp.ParamStoreKeyVersionParams)},
+		// staking.BondDenom
+		{stakingtypes.ModuleName, string(stakingtypes.KeyBondDenom)},
+		// staking.MaxValidators
+		{stakingtypes.ModuleName, string(stakingtypes.KeyMaxValidators)},
+		// staking.UnbondingTime
+		{stakingtypes.ModuleName, string(stakingtypes.KeyUnbondingTime)},
```

Comment:
[blocking] this adds more params to the list of blocked params which is potentially something we need to do so that there isn't a divergence between what is stated in params.md and what is actually modifiable. However, it's not clear if these need to be added without appropriate test coverage.

IMO we should tackle this in a few steps:
1. Write a table-driven test with test cases for each param listed in params.md. The test should attempt to modify the param and see if it was actually modified. Each test case should have a boolean field that indicates if modification was expected to happen (i.e. that param is governance modifiable) or not happen (i.e. that param is not governance modifiable).
2. Once we have the test above, we can compare the test results to params.md. If any divergences exist, create a GH issue with the divergence so that we can get alignment on whether the param should be governance modifiable.
3. [Optional] If there are any params that are governance modifiable and shouldn't be, we may need to add them to this BlockedParams list.